### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by NikonD850** [csprh/MAMAT#2]
Nice work! I'm wonder if you can provide pretrained models in your convenience. Thanks in advance!

**Comment by LCH0528** [comment]
Could you please show me your email address, such as QQ or school email? After presenting it, I will send the file to you via email.

**Comment by LCH0528** [comment]
If you are the first author, Professor Paul Hill, I have also sent you an email before.
By the way, actually, the author of the Deturb paper, Tyreal-Izual (github name), also asked a similar question. I have sent the materials to him as well. You can also contact him. However, it's not clear if it's due to the QQ email itself. It seems that he didn't receive the code email sent over, but received the compressed file.

**Comment by NikonD850** [comment]
> Could you please show me your email address, such as QQ or school email? After presenting it, I will send the file to you via email.

Greetings!
Here is my email: nikoncopd850@gmail.com
Thanks!

**Comment by LCH0528** [comment]
Are you the author of the paper MAMAT?&nbsp;Who exactly is it?

**Comment by NikonD850** [comment]
> Are you the author of the paper MAMAT?&nbsp;Who exactly is it?

Exactly no! My issue is not a reply to your issue #1 , and I'm seeking for the origin pre-trained model.

**Comment by LCH0528** [comment]
Ok, actually, I'm also looking for the weight files in the project, but I can't get them either (I asked the author of Deturb before, and he said he can't find that file for the time being). The ones I sent him and you in the email are just the model code files I trained myself (the dataset is from TMT, which is different from his). And the training failed (the effect was not as good as the pure unet3d in the TMT paper). It's just that you want it for your reference and analysis.&nbsp;
In conclusion, I can't get the weight files in either the MAMAT or Deturb projects. I'm sorry.


译文：
好吧，其实我也在找项目中的权重文件，但也是要不到（之前问过Deturb的作者的，他说他现在那个文件暂时找不到了），我发给他和您邮件中的只是我自己拿模型代码文件训练的（数据集是TMT的，跟他的不一样），且训练失败了（效果没有TMT论文中纯粹的unet3d好），只是你们想要给你们参考分析罢了。
总之，MAMAT和Deturb项目中的权重文件我都要不到，不好意思。

**Comment by LCH0528** [comment]
If you have found the corresponding weight file of the author,&nbsp; please send it to my QQ email.



---Original---
From: ***@***.***&gt;
Date: Wed, Jul 16, 2025 10:36 AM
To: ***@***.***&gt;;
Cc: "Chenhao ***@***.******@***.***&gt;;
Subject: Re: [csprh/MAMAT] Pretrained Model (Issue #2)


NikonD850 left a comment (csprh/MAMAT#2)
  
Are you the author of the paper MAMAT?&nbsp;Who exactly is it?
  
Exactly no! My issue is not a reply to your issue #1 , and I'm seeking for the origin pre-trained model.

**Comment by ZYF200014** [comment]
Hello, could you please send me a pre-trained model from TMT? Here is my email address: zyf1312022@163.com

**Comment by LCH0528** [comment]
Sorry, if it's the weight file of the deformable U-Net that I trained using the TMT dataset as mentioned in our previous chat records, it might not be available anymore (or it might have been deleted because I didn't use it later and instead used the weights from the original author of TMT).
If it is by the original author of TMT, the link is:
https://github.com/xg416


译文：不好意思，如果是之前聊天记录中提到的我自己使用TMT数据集训练的可变形U-Net权重文件，可能后来找不到了（也可能被删掉了，因为后面我没用到，就用TMT原作者的权重）。如果是TMT原作者的，链接（3个项目）：
https://github.com/xg416
         原始邮件
         
       
发件人：ZYF200014 ***@***.***&gt;
发件时间：2025年11月15日 12:04
收件人：csprh/MAMAT ***@***.***&gt;
抄送：Chenhao Li ***@***.***&gt;, Comment ***@***.***&gt;
主题：Re: [csprh/MAMAT] Pretrained Model (Issue #2)



ZYF200014 left a comment (csprh/MAMAT#2)

Hello, could you please send me a pre-trained model from TMT? Here is my email address: ***@***.***

**Comment by ZYF200014** [comment]
Hello , do you have a copy of the best model saved after TMT training? Due to resource allocation issues in the laboratory, it may take me a month to complete the run here. If you have it, is it convenient for you to share? If it's inconvenient, could you please let me know the metric values trained on the ATSyn_dynamic dataset? Thank you, you can send it to my email: zyf1312022@163.com Wishing you a happy life!

**Comment by LCH0528** [comment]
Sorry! As I mentioned before, the weight file is missing. However, even if the weight file is found, it is not recommended to use it: I remember that the training file at that time was the weights of a deformable U-net that I trained using the TMT dataset, but the result was not good. Furthermore, in fact, it seems that this weight was actually used in transfer learning: both the TMT and the ATSyn_dynamic datasets utilized it. At that time, the laboratory conditions were also poor. Later, no improvements were made in this area. Instead, efforts were focused on other aspects, and cloud training (AUTODL) was adopted (it cost nearly 10,000, and the school may not cover the expense). Furthermore, I had previously asked the original author for it, but he said he had graduated and left, so the weight file was no longer available either. There's nothing I can do. I'm really sorry.




译文：
抱歉！之前讲过了，权重文件不见了。不过，就算找到了权重文件，也不建议使用：我记得当时的训练文件是自己用TMT数据集训练的可变形U-net的权重，但效果不好；并且，实际上，好像这个权重应该是用到了迁移学习：TMT和ATSyn_dynamic dataset都用到了。我当时实验室条件也差，后面是没有在这个方面改进，改了其他方面，再加上云端训练（AUTODL）搞成的（花了快10000，学校不一定报销）。此外，之前问了原作者要过，他说他已经毕业了，离开了，那个权重文件也没了，也要不到了。没办法了，真不好意思。
         原始邮件
         
       
发件人：ZYF200014 ***@***.***&gt;
发件时间：2025年11月15日 19:47
收件人：csprh/MAMAT ***@***.***&gt;
抄送：Chenhao Li ***@***.***&gt;, Comment ***@***.***&gt;
主题：Re: [csprh/MAMAT] Pretrained Model (Issue #2)



ZYF200014 left a comment (csprh/MAMAT#2)

Hello , do you have a copy of the best model saved after TMT training? Due to resource allocation issues in the laboratory, it may take me a month to complete the run here. If you have it, is it convenient for you to share? If it's inconvenient, could you please let me know the metric values trained on the ATSyn_dynamic dataset? Thank you, you can send it to my email: ***@***.***&nbsp;Wishing you a happy life!

**Comment by ZYF200014** [comment]
Thank you for your reply, and I wish you all the best！


派大星的海棉裤
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "csprh/MAMAT"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年11月15日(星期六) 晚上9:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [csprh/MAMAT] Pretrained Model (Issue #2)



LCH0528 left a comment (csprh/MAMAT#2)
 Sorry! As I mentioned before, the weight file is missing. However, even if the weight file is found, it is not recommended to use it: I remember that the training file at that time was the weights of a deformable U-net that I trained using the TMT dataset, but the result was not good. Furthermore, in fact, it seems that this weight was actually used in transfer learning: both the TMT and the ATSyn_dynamic datasets utilized it. At that time, the laboratory conditions were also poor. Later, no improvements were made in this area. Instead, efforts were focused on other aspects, and cloud training (AUTODL) was adopted (it cost nearly 10,000, and the school may not cover the expense). Furthermore, I had previously asked the original author for it, but he said he had graduated and left, so the weight file was no longer available either. There's nothing I can do. I'm really sorry. 
 
 
 
 
 译文： 
 抱歉！之前讲过了，权重文件不见了。不过，就算找到了权重文件，也不建议使用：我记得当时的训练文件是自己用TMT数据集训练的可变形U-net的权重，但效果不好；并且，实际上，好像这个权重应该是用到了迁移学习：TMT和ATSyn_dynamic dataset都用到了。我当时实验室条件也差，后面是没有在这个方面改进，改了其他方面，再加上云端训练（AUTODL）搞成的（花了快10000，学校不一定报销）。此外，之前问了原作者要过，他说他已经毕业了，离开了，那个权重文件也没了，也要不到了。没办法了，真不好意思。 
          原始邮件 
           
         
 发件人：ZYF200014 ***@***.***&amp;gt; 
 发件时间：2025年11月15日 19:47 
 收件人：csprh/MAMAT ***@***.***&amp;gt; 
 抄送：Chenhao Li ***@***.***&amp;gt;, Comment ***@***.***&amp;gt; 
 主题：Re: [csprh/MAMAT] Pretrained Model (Issue #2) 
 
 
 
 ZYF200014 left a comment (csprh/MAMAT#2) 
 
 Hello , do you have a copy of the best model saved after TMT training? Due to resource allocation issues in the laboratory, it may take me a month to complete the run here. If you have it, is it convenient for you to share? If it's inconvenient, could you please let me know the metric values trained on the ATSyn_dynamic dataset? Thank you, you can send it to my email: ***@***.***&amp;nbsp;Wishing you a happy life!

**Comment by pui-nantheera** [comment]
Can you please check if this works: [pretrained model](https://uob-my.sharepoint.com/:u:/g/personal/eexna_bristol_ac_uk/Ee-DQffcT2BMsqFkMpd7_tQBk1a3jkcxFFnNejPjbMxT3Q?e=Q4GriS).

Thanks.

**Comment by LCH0528** [comment]
Sorry, I'm currently in my senior year of study and have been quite busy recently. I might not have time to help you with additional training on other things. I have downloaded and saved the weights for you. I've sent it to you. You can handle it by yourself. If the conditions are not sufficient, cloud-based training (such as AutoDL) can be adopted.


译文：抱歉，本人目前研三了，最近比较忙，可能没空帮你额外训练其他东西了。我帮你下载下来权重了，发给你了，你自己处理吧。如果条件不足，可以云端训练（AutoDL等）。


         原始邮件
         
       
发件人：pui-nantheera ***@***.***&gt;
发件时间：2025年11月16日 20:24
收件人：csprh/MAMAT ***@***.***&gt;
抄送：Chenhao Li ***@***.***&gt;, Comment ***@***.***&gt;
主题：Re: [csprh/MAMAT] Pretrained Model (Issue #2)



pui-nantheera left a comment (csprh/MAMAT#2)

Can you please check if this works: pretrained model.

Thanks.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.


 	
 	 		 			从QQ邮箱发来的超大附件 	
 	 		  				 					 						 							 						 					
 					 						 							turb_model_best.pth 							 (32.7MB, 2026年2月14日 20:49) 						 						 							进入下载页面 							：https://wx.mail.qq.com/ftn/download?func=3&k=99cd0a324e5b24e2abbe4b3237643530e4c46e30356435304e1d08541a5151030205520750490256045748060d53061d040150021856050854535c57575c54015e2165464016576f0b5d0157593b575515464b42410c2153c483772157763ad9d8f6a159a95ebcee09dd963935643530663265&key=99cd0a324e5b24e2abbe4b3237643530e4c46e30356435304e1d08541a5151030205520750490256045748060d53061d040150021856050854535c57575c54015e2165464016576f0b5d0157593b575515464b42410c2153c483772157763ad9d8f6a159a95ebcee09dd963935643530663265&code=f2e25d50&from=

**Comment by NikonD850** [comment]
> Can you please check if this works: [pretrained model](https://uob-my.sharepoint.com/:u:/g/personal/eexna_bristol_ac_uk/Ee-DQffcT2BMsqFkMpd7_tQBk1a3jkcxFFnNejPjbMxT3Q?e=Q4GriS).
> 
> Thanks.

Thanks @pui-nantheera ! This pretrained model works well. Please consider to add the link in README.md @csprh which will help a lot.
